### DISCUSSION TITLE
Sunbird RC - Recorded Demo

### DISCUSSION THREAD

**Discussion by manukalia29** [Sunbird-RC/community#687] [Questions (FAQs)]
@Renuka-S, Is there a recorded demo or a demo instance available for RC?

**Comment by parthlawate** [comment]
@manukalia29 there is a bunch of videos linked from some of the reference solutions. Also on reference solution level there are some live demos..  eg links https://docs.sunbirdrc.dev/reference-solutions/organ-registries

**Reply by manukalia29** [reply]
Thanks @parthlawate Is there a demo video available that showcases the core RC stack including the UI, and provides an end-to-end demonstration from schema definition to customizing UI?

**Reply by parthlawate** [reply]
No that isnt there. The videos we have currently are only of reference solutions. This is a good idea we should make one end to end. 
1. Installation
2. Configuring backend with Schema 
3. Configuring frontend with schema  

This would also be super useful for the learning path @coolbung defined here https://github.com/orgs/Sunbird-RC/discussions/676

**Reply by manukalia29** [reply]
@parthlawate  Can you align someone from the team to create the demo video?

**Reply by parthlawate** [reply]
@Renuka-S can you please work with someone in the team to create this ? Ideally we want someone to follow the docs & do the above steps & record a video doing that.

**Comment by manukalia29** [comment]
Thanks @parthlawate, how much time it will take to create a demo instance?

**Reply by parthlawate** [reply]
@manukalia29 typically when RC is deployed its for a certain use case. If its a greenfield demo of a sample schema should be doable in a day..
